### ISSUE TITLE
(reticulate) handle blank lines in python code chunks sent to `reticulate::repl_python`

### ISSUE THREAD

**Issue by jmbuhr** [jmbuhr/quarto-nvim-kickstarter#97]
otherwise blank lines cause e.g. classes or methods to end prematurely. Need some form  of bracketed paste.

**Comment by akthe-at** [comment]
will this be solved with the new repl that is coming in python 3.13?

**Comment by jmbuhr** [comment]
No, the issue is in a different place, but that's still a thing to look out for in the future, thanks for the heads up.
